From 5c0b6653d0fa500b5ad3a65f8e58c97517f2d6bd Mon Sep 17 00:00:00 2001
From: Philipp Schrader <philipp.schrader@gmail.com>
Date: Wed, 21 Feb 2024 17:26:27 -0800
Subject: [PATCH] Fix resolving files

I don't really know what the underlying problem is, but returning a
File instead of a path is causing us grief.
---
 rollup/private/rollup.bzl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rollup/private/rollup.bzl b/rollup/private/rollup.bzl
index c1634cf..8cd5407 100644
--- a/rollup/private/rollup.bzl
+++ b/rollup/private/rollup.bzl
@@ -96,7 +96,7 @@ def _resolve_js_input(f, inputs):
     for i in inputs:
         if i.extension == "js" or i.extension == "mjs":
             if _no_ext(i) == no_ext:
-                return i
+                return i.short_path
     fail("Could not find corresponding javascript entry point for %s. Add the %s.js to your deps." % (f.path, no_ext))
 
 def _rollup_outs(sourcemap, name, entry_point, entry_points, output_dir):
